### PR TITLE
API Gateway May 23 Update Important changes section

### PR DESCRIPTION
**Pull request by kmercierwhite**
http://pr-2647.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_relnotes/20230530_apimgr_relnotes/#update-your-custom-helm-chart-values-file-if-using-custom-certificates

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | db5bb4a |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-2647.axway-open-docs.opendocs-builder.pcloud.axway.int |

File: content/en/docs/apim_relnotes/20230530_apimgr_relnotes.md [hunk 1]
```diff
@@ -88,6 +88,52 @@ Note that the previously documented plural form (for example, `seconds`) is no l
...
+  domainId: "prod"
+```
+
+### Update your custom Helm chart values file to accept general conditions
+
+The parameter(s) generalConditions.accept must be set for each image that is pulled from the Axway repository. The parameter is described in the supplied values.yaml file.
+
+For example, in the Admin Node Manager image it must be added as in the example below under the image section.
+
+```yaml
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20230530_apimgr_relnotes.md:110]: @kmercierwhite here the link for you to use

For more information, see [Update your custom Helm chart values file to accept general conditions](docs/apim_installation/apigw_containers/deployment_flows/axway_image_deployment/helm_runtime/#acceptance-of-general-conditions-for-license-and-subscription-services).

File: content/en/docs/apim_relnotes/20230530_apimgr_relnotes.md [hunk 2]
```diff
@@ -88,6 +88,52 @@ Note that the previously documented plural form (for example, `seconds`) is no l
 
 Data Maps cannot be configured or updated in Policy studio in this release. For more information, see [Known issues](#data-maps-cannot-be-created-or-configured-in-policy-studio) section.
 
+### Update your custom Helm chart values file if using custom certificates
+
+If using custom certificates the following change is required when using Axway images:
+
+In the `global` section of your custom values file, add the new field domainId and give it a value for the domain id. This value should match the domain id used in the creation of your custom certificates
+
+For example:
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20230530_apimgr_relnotes.md:97]: For more informations, see [Add a customized domain certificate configuration](/docs/apim_installation/apigw_containers/deployment_flows/axway_image_deployment/helm_deployment/#add-a-customized-domain-certificate-configuration)

File: content/en/docs/apim_relnotes/20230530_apimgr_relnotes.md [hunk 3]
```diff
@@ -88,6 +88,52 @@ Note that the previously documented plural form (for example, `seconds`) is no l
...
+      accept: “yes”
+```
+
+This should be done similarly for the aga(analytics), apimgr and apitraffic images.
+
+### The Axway API Gateway Helm chart has been updated to remove any ambiguity around setting of service ports
+
+In the custom values file please ensure that any overwrites to service ports match exactly the layout in the default values file:
+
+For example:
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20230530_apimgr_relnotes.md:125]: 'For more information, see.....'  --- I'm not sure where this is located in the product's documentation.

http://master.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_installation/apigw_containers/

Comment (andreamussap) [reply content/en/docs/apim_relnotes/20230530_apimgr_relnotes.md:103]: Note to self - fback from Mary: " not knowing if there is a reference to the default values file in the documentation and you could be right - maybe there isn't.  To resolve this you could replace the word 'helm' with the word 'default' in this section i.e "To be able to view the default values.yaml file" 

/docs/apim_installation/apigw_containers/deployment_flows/axway_image_deployment/helm_deployment/#fetch-the-helm-chart-to-examine-the-values-file

**Pull request merged**